src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -281,5 +288,28 @@ private static string GetExpectedRedirectUri(string clientId)
         {
             return $"ms-appx-web://microsoft.aad.brokerplugin/{clientId}";
         }
+
+        /// <summary>
+        /// Validate common auth params
+        /// </summary>
+        /// <param name="authenticationRequestParameters"></param>
+        /// <param name="logger"></param>
+        /// <exception cref="MsalClientException"></exception>
+        public static void ValidateAuthParams(
```

Comment:
This is used in only one place in the same class. Please make it private. (or write unit test  :))